dotcom-rendering/src/web/components/FilterButton.importable.tsx

```diff
@@ -71,7 +71,9 @@ const valueStyles = css`
 const Label = ({ value, count }: LabelProps) => (
 	<>
 		<span css={valueStyles}>{value}</span>{' '}
-		{!!count && <span css={countStyles}>({count})</span>}
+		{count !== undefined && count !== 0 && (
+			<span css={countStyles}>({count})</span>
+		)}
```

Comment:
It is quite odd that this is required at all… what does a value of `0` even mean? How can you you have a selected topic without a count?

I think that it would be best to fallback to a default value of `0` for `count` and check if it’s a strictly positive integer:

```tsx
const Label = ({ value, count = 0 }: LabelProps) => (
	<>
		<span css={valueStyles}>{value}</span>{' '}
		{count > 0 && (
			<span css={countStyles}>({count})</span>
		)}
	</>
);
```

Reply:
Yeah I too am confused by the logic ... I didn't want. to refactor too much of edex's code, but I'll go with your suggestion :)

Reply:
None of these comments are blockers… but it’s hard to know what to include in a refactor. Maybe @guardian/editorial-experience can chime in directly?